### PR TITLE
BXMSPROD-572: removed obsolete deps

### PR DESCRIPTION
**Pull request by mbiarnes**
@mareknovotny the [test build](https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/custom/job/mbiarnes/job/mbiarnes_kieparent/2/console) was OK but UNSTABLE.
There only failed one [test](https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/custom/job/mbiarnes/job/mbiarnes_kieparent/2/testReport/) that also failed in [daily build master](https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/daily-build/job/daily-build-pipeline-master/30/testReport/)

### CODE REVIEW

**Comment by mbiarnes** [comment]
Jenkins execute full downstream build

File: pom.xml [hunk 1]
```diff
@@ -3478,16 +3150,6 @@
...
-      <dependency>
-        <groupId>org.apache.poi</groupId>
-        <artifactId>poi-ooxml-schemas</artifactId>
-        <version>${version.org.apache.poi}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.apache.poi</groupId>
-        <artifactId>poi-scratchpad</artifactId>
-        <version>${version.org.apache.poi}</version>
-      </dependency>
```
Comment (mareknovotny) [inline pom.xml]: we need  to ask @ge0ffrey @manstis if these managed deps declaration are ok to remove

File: pom.xml [hunk 2]
```diff
@@ -3587,11 +3223,6 @@
         <artifactId>batik-ext</artifactId>
         <version>${version.org.apache.xmlgraphics.batik}</version>
       </dependency>
-      <dependency>
-        <groupId>org.apache.xmlgraphics</groupId>
-        <artifactId>batik-extension</artifactId>
-        <version>${version.org.apache.xmlgraphics.batik}</version>
-      </dependency>
       <dependency>
```
Comment (mareknovotny) [inline pom.xml]: @tsurdilo is it ok to remove this?

File: pom.xml [hunk 3]
```diff
@@ -3807,11 +3420,6 @@
         <artifactId>jetty-servlets</artifactId>
         <version>${version.org.eclipse.jetty}</version>
       </dependency>
-      <dependency>
-        <groupId>org.eclipse.jetty</groupId>
-        <artifactId>jetty-util</artifactId>
-        <version>${version.org.eclipse.jetty}</version>
```
Comment (yurloc) [inline pom.xml]: Why changing the alphabetical order?

Comment (mbiarnes) [reply pom.xml]: @yurloc first I removed the eclipse deps - saw that tests were failing - restored them again - and here I must disregard the alphabetical order - we can change this later on

**Comment by mbiarnes** [comment]
Jenkins execute full downstream build

**Comment by mbiarnes** [comment]
Jenkins execute full downstream build

**Comment by mbiarnes** [comment]
@mareknovotny green? Lets merge it before we get new conflicts.

**Pull request merged**